app/utility/packers/upx.py

```diff
@@ -0,0 +1,33 @@
+import os
+import asyncio
+
+name = 'upx'
```

Comment:
please ensure behavior when upx is not installed

Reply:
Added in a dependency check and if it fails the packer is not loaded.

Though this would bring up the question of, what would the expected behavior be if the packer is requested by an ability, and it is not available?  Should we fail the payload, or should we pass on the payload unpacked as it is not able to be.